ISISComputingGroup/EPICS-ioc
# Ticket1404 lakeshore336 ioc

### Description of work

Added IOC for Lakeshore336 Temperature Controller in LKSH336/
Note that and old, test version of the IOC (which wasn't running anything) in LS336/ has been deleted

When testing on the real thing, you'll need to set your `globals.txt file` to have the correct address for the device. The device has a bug which means it doesn't display its IP correctly, but you can set the macro to the device name: `ls336-1` (`LKSH336_01__IPADDR=ls336-1` in globals.txt)


### To test

Fixes Ticket ISISComputingGroup/IBEX#1404
See ticket for list of related pull requests

### Acceptance criteria

* The IOC communicates correctly with the actual device
* The IOC runs in simulation mode
* The IOC runs with the associated emulator
* The IOC LKSH336_02 works too!


---

#### Code Review


### Functional Tests


### Final steps
